model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -57,16 +57,22 @@ data class OrtConfiguration(
     val advisor: Map<String, AdvisorConfiguration>? = null
 ) {
     companion object {
+        /**
+         * The name of the embedded reference configuration resource.
+         */
+        const val REFERENCE_RESOURCE = "/reference.conf"
```

Comment:
I am not sure whether the reference configuration, which is actually not a valid configuration, should be mentioned here at this prominent place. IIUC, production code should never read this configuration.

Reply:
I'd be fine with moving this to `ConfigCommand`. Any other opinions by anyone? My thinking was that if ORT ever got a web frontend in addition to the CLI frontend, the web frontend might want to display the reference config as part of docs / some help screen, and thus I've placed this constant in library code for use with the API.

Reply:
This is obsolete now as the constant was removed.